docs/usage/configuration-options.md

```diff
@@ -2754,3 +2754,17 @@ To disable the vulnerability alerts functionality completely, configure like thi
   }
 }
 ```
+
+<!-- prettier-ignore -->
+!!! tip
+    We strongly recommend you have good test coverage before you automerge vulnerability alerts.
```

Comment:
The emphasis for this sentence is on "automerge", not on "vulnerability alerts", right? We want to encourage users to reel in security fixes ASAP (benefiting the community), but that becomes risky when the code coverage is low. Can we rephrase to get that point across?